src/title/ab_glyph_renderer.rs

```diff
@@ -80,8 +80,8 @@ impl AbGlyphTitleText {
 
         let glyphs = self.layout(&font);
         let last_glyph = glyphs.last()?;
-        let width = (last_glyph.position.x + font.h_advance(last_glyph.id)).ceil() as u32;
-        let height = font.height().ceil() as u32;
```

Comment:
A week from now, noone will remember why there is + 2 here.
Let's add a short comment along the lines of `+ 2 because ab_glyph likes to draw outside of its area, so we add 1px border around the pixmap`

Reply:
Good point 😅